app_test.go

```diff
@@ -1115,8 +1111,8 @@ func TestApp_ParseSliceFlags(t *testing.T) {
 		}
 		return true
 	}
-	var expectedIntSlice = []int{22, 80}
-	var expectedStringSlice = []string{"8.8.8.8", "8.8.4.4"}
+	expectedIntSlice := []int{22, 80}
+	expectedStringSlice := []string{"8.8.8.8", "8.8.4.4"}
```

Comment:
I think you have auto-formatting turned on in your code editor. Could you undo changes like these that are unrelated to the PR? That would make the diff clearer for the reviewers. Thanks!